basic/src/test/java/org/jboss/hal/testsuite/test/configuration/infinispan/CacheContainersTestCase.java

```diff
@@ -41,12 +40,14 @@
     private static final String JNDI_NAME = "java:/" + CACHE_CONTAINER_NAME;
 
     private static final OnlineManagementClient client = ManagementClientProvider.createOnlineManagementClient();
-    private final Operations ops = new Operations(client);
-    private final Administration adminOps = new Administration(client);
+    private static final Operations ops = new Operations(client);
+    private static final Administration adminOps = new Administration(client);
```

Comment:
I don't understand why `ops` and `adminOps` have to be `static` suddenly but have no problem with it.